zerver/two_factor_migrations/0001_initial.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+from django.conf import settings
+import django.core.validators
```

Comment:
It seems that `two_factor` doesn't ship with a complete set of migrations. I can create fork.